src/cozmo/robot.py

```diff
@@ -1700,9 +1697,9 @@ def roll_cube(self, target_object, approach_angle=None, check_for_object_on_top=
         '''Tells Cozmo to roll a specified cube object.
 
         Args:
-            target_object (:class:`cozmo.objects.LightCube`): The destination object.
-            approach_angle (:class:`cozmo.util.Angle`): The angle to approach the cube from.
-            check_for_object_on_top (bool): only roll the cube if there isn't a cube on top of it
+            target_object (:class:`cozmo.objects.LightCube`): The cube to roll.
+            approach_angle (:class:`cozmo.util.Angle`): The angle to approach the cube from.   For example, 180 will cause cozmo to drive past the cube and approach it from behind.
+            check_for_object_on_top (bool): only roll the cube if there isn't a cube on top of it.  If there is the cube will be ignored.
```

Comment:
I don't think "If there is the cube will be ignored."  is correct, suggest changing to e.g. "If there is a cube on top, and check_for_object_on_top is True, then Cozmo will ignore the action"